.circleci/config.yml

```diff
@@ -28,7 +28,7 @@ executors:
   tfsec:
     resource_class: small
     docker:
-      - image: circleci/python:latest
+      - image: cimg/base:current
```

Comment:
We were using a deprecated image here introduced in PR #183. Update this to just use the base convenience image since we don't need Python.